Source/WebCore/platform/text/cocoa/LocalizedDateCache.mm

```diff
@@ -133,13 +131,14 @@ static void _localeChanged(CFNotificationCenterRef, void*, CFStringRef, const vo
 
 static float calculateMaximumWidthForWeek(const MeasureTextClient& measurer)
 {
-    std::array<float, 10> numLengths = { };
-    for (int i = 0; i < 10; i++)
+    std::array<float, 10> numLengths;
+    for (int i = 0; i < 10; ++i)
```

Comment:
Can indexedRange() be used here? (I ask because I am not clear on how well it handles mutable references). (Also, all this code is bananas).

Reply:
I actually don't know if indexedRange() handles mutable references well.